web/src/ChatListPage.js

```diff
@@ -85,19 +85,19 @@ class ChatListPage extends BaseListPage {
             },
           });
         } else {
-          Setting.showMessage("error", `Failed to add Chat: ${res.msg}`);
+          Setting.showMessage("error", `${i18next.t("chat:Failed to add Chat")}: ${res.msg}`);
```

Comment:
Use this way:

https://github.com/casdoor/casdoor/blob/86c10fe0abbf4cbcfec2b75663c38a8a1de72ec5/web/src/locales/zh/data.json#L268-L276

It means to use "Failed to add", not "Failed to add XXX". This will dramatically reduce string count.

Conflicts also need to be resolved